scripts/substitute_article_urls.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Update to work with translations; may require changing the urls.yaml
```

Comment:
Are we even using the functionality in this file anymore?  It looks like we're just always hard-linking to innersourcecommons.org.

Reply:
Good point, I can't think of any requirements we have that this would help, unless we're planning to upload to other platforms or re-uploading to O'Reilly.

Reply:
Not that I know of.